source/fuzz/transformation_replace_id_with_synonym.cpp

```diff
@@ -140,6 +140,51 @@ bool TransformationReplaceIdWithSynonym::IsAgnosticToSignednessOfOperand(
     case SpvOpUGreaterThanEqual:
     case SpvOpSGreaterThanEqual:
       return true;
+
+    case SpvOpAtomicStore:
+    case SpvOpAtomicExchange:
+    case SpvOpAtomicIAdd:
+    case SpvOpAtomicISub:
+    case SpvOpAtomicSMin:
+    case SpvOpAtomicUMin:
+    case SpvOpAtomicSMax:
+    case SpvOpAtomicUMax:
+    case SpvOpAtomicAnd:
+    case SpvOpAtomicOr:
+    case SpvOpAtomicXor:
+    // TBD(To Be Discussed),
+    // https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/EXT/SPV_EXT_shader_atomic_float_add.asciidoc#modifications-to-the-spir-v-specification-version-15
+    case SpvOpAtomicFAddEXT:  // Capability (AtomicFloat32AddEXT,
+                              // AtomicFloat64AddEXT)
+      assert(use_in_operand_index != 0 && "Forbidden.");
+
+      if (opcode == SpvOpAtomicFAddEXT) {
+        // Would you like to check Capability here?
+      }
+      return use_in_operand_index == 1 || use_in_operand_index == 2;
+
+    case SpvOpAtomicCompareExchange:
+    // Deprecated, missing after version 1.3.
+    // Would you like to remove it?
+    case SpvOpAtomicCompareExchangeWeak:  // Capability (Kernel)
+      assert(use_in_operand_index != 0 && "Forbidden.");
+      // This opcode is Deprecated!
+      if (opcode == SpvOpAtomicCompareExchangeWeak) {
+        // Would you like to check Capability here?
+      }
```

Comment:
Same as above, there's no need for this check. If the module contains an OpAtomicCompareExchangeWeak instruction then we should handle signedness of its operands appropriately.

Reply:
OK, Done.